react-props/my-app/src/ButtonContainer.tsx

```diff
@@ -0,0 +1,11 @@
+export function ButtonContainer({ btnContainer }: btnProp) {
+  return (
+    <div>
+      <button>{btnContainer}</button>
+    </div>
+  );
+}
+
+type btnProp = {
```

Comment:
Type names should start with uppercase.

I think the code reads nicer if the props type is above the component rather than below it.